crosschain/src/main/java/org/hyperledger/besu/crosschain/core/keys/CrosschainKeyManager.java

```diff
@@ -100,6 +100,22 @@ public void init(final BigInteger sidechainId, final SECP256K1.KeyPair nodeKeys)
     this.thresholdKeyGenContract.init(nodeKeys);
   }
 
+  /**
+   * This method is used to restore the key data from the persistent store when the node restarts.
+   *
+   * @param activeKeyGenerations Current credentials
```

Comment:
Is this correct?

Reply:
Actually, I am not sure. How can we best describe activeKeyGenerations and credentials? @drinkcoffee

Reply:
I misread your comment previously. I thought it was for function name.